aea/configurations/base.py

```diff
@@ -289,6 +292,157 @@ def dependencies_to_json(dependencies: Dependencies) -> Dict[str, Dict]:
     return result
 
 
+class CertRequest:
```

Comment:
Needs to be moved to helpers class. Please review https://github.com/fetchai/agents-aea/pull/2050 before continuing (in particular, `AgentRecord`)